### PR TITLE
 Updated External Lib Apache Commons-Cli build process

### LINKED ISSUE #616: Updated build process for apache commons-cli plugin
Same as in #614

### PR DESCRIPTION
**Pull request by ghost**
Same as in #615 which should be merged before.

Closes #616

### CODE REVIEW

File: de.dlr.sc.virsat.external.lib.commons.cli/about.html [hunk 1]
```diff
@@ -32,13 +32,25 @@ <h3>Third Party Content</h3>
 terms and conditions of use. Third party licenses are aggregated in the &quot;about_files&quot; folder of this plugin.
 </p>
 
-<h4>Apache Commons CLI</h4>
+<h4>Directly Maven Included Thrid Party Libraries</h4>
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.external.lib.commons.cli/about.html:35]: Typo: "Thrid" -> "Third"
Do we have this typo in other places as well? :D

**Review by SaMuellerDLR — CHANGES_REQUESTED** [review]
Only a minor typo.

File: de.dlr.sc.virsat.external.lib.commons.cli/about.html (hunk 1 above)
Comment (PhilMFischer) [inline de.dlr.sc.virsat.external.lib.commons.cli/about.html:35]:
```suggestion
<h4>Directly Maven Included Third Party Libraries</h4>
```

Comment (PhilMFischer) [reply de.dlr.sc.virsat.external.lib.commons.cli/about.html:35]: @SaMuellerDLR As such?

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Review by franzTobiasDLR — APPROVED** [review]

**Comment by franzTobiasDLR** [comment]
What was the trick again to trigger the check task-list-completed to update? I just create a new task and marked it as complete but it didn't change...

**Comment by SaMuellerDLR** [comment]
@franzTobiasDLR According to the dev https://github.com/stilliard/github-task-list-completed/issues/7 they host the app on glitch and just like in this older linked issue, the glitch platform seems to be having some major incident right now https://status.glitch.com/

**Pull request merged**